src/main/java/tds/content/TdsContentServiceApplication.java

```diff
@@ -1,4 +1,4 @@
-package org.opentestsystem.delivery;
+package tds.content;
```

Comment:
We can go either way, but yes I was just being consistent with other tds apps (tds.assessment, tds.exam, etc)